testslide/runner.py

```diff
@@ -180,6 +180,42 @@ def print_cyan(self, *values, **kwargs):
 
 
 class FailurePrinterMixin(ColorFormatterMixin):
+    def _print_stack_trace(self, exception: BaseException, cause: int) -> None:
```

Comment:
Maybe rename `cause` to `cause_level` or similar? I thought it was a boolean kinda thing until I reached the end of the function :man_shrugging: